dataplane/vppagent/conf/supervisord/supervisord-dev.conf

```diff
@@ -29,7 +38,7 @@ command=/bin/vppagent-dataplane
 startsecs=0
 autorestart=false
 redirect_stderr=true
-priority=3
+priority=4
```

Comment:
Curious... why priority four here?

Reply:
Ah, this one actually doesn't play a role. 

Historically I'was trying to resolve one issue related to health-probing via sub-process prioritization.
When GDB attaches to vpp, the vpp usually freezes for about 0.7s. This is enough to cause intermittent issue with agent complaining on non-responsive vpp and exiting with error. I've [increased](https://github.com/networkservicemesh/networkservicemesh/pull/762/files/c0c9a6c7e5eb532d892a566c2d35a5875d70c0b5#diff-d9a7275054fd2ab8910d32d4a5f0762aR28) reply timeout in dev-image to resolve this issue, but forgot to rollback changes in `supervisord-dev.conf`. 

Currently it shouldn't affect anything.